lib/cli/commands/discovernodes.ts

```diff
@@ -2,13 +2,13 @@ import { callback, loadXudClient } from '../command';
 import { Arguments } from 'yargs';
 import { DiscoverNodesRequest } from '../../proto/xudrpc_pb';
 
-export const command = 'discovernodes <peer_pub_key>';
+export const command = 'discovernodes <node_key>';
 
 export const describe = 'discover nodes from a specific peer';
 
 export const builder = {
-  peer_pub_key: {
-    description: 'nodePubKey of the peer to discover nodes from',
+  node_key: {
```

Comment:
I'd be a bit hesitant renaming to `node_key` in that it may be confusing and is inconsistent with lnd's naming for their similar concept, which is almost always referred to as "pubkey" or "public key" from what I can tell. It would also be inconsistent with our API naming.

At a minimum I think we should mention "public" in the description if we keep `node_key` as the arg name.

Reply:
I'm a bit confused. Above we agreed to move everything to "node key", especially in the descriptions. So what exactly what do you want to change? Call all "node key" "node public key" in the `description:` fields?

Reply:
Sorry I just realized I gave conflicting feedback by suggesting "node key" myself as an alternative further up. I suppose it's fine, although I still slightly prefer "node pubkey" and "node_pubkey".

Reply:
Did stick with "node key" and "node_key".

Reply:
We also need to change line 18 in this file `peer_pub_key->node_key`.